component/src/components/composed/__tests__/chart-options-panel.test.tsx

```
@@ -3,11 +3,17 @@ import { describe, it, expect, vi } from "vitest";
 import { ChartOptionsPanel } from "../chart-options-panel";
 import { getChartOptions } from "../chart-options-schema";
 
+/** Expand all collapsed category sections so their content is in the DOM. */
+function expandAllCategories() {
+  screen.getAllByRole("button", { expanded: false }).forEach((btn) => fireEvent.click(btn));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)